backend/substrapp/tasks/tasks.py

```diff
@@ -1019,8 +1019,9 @@ def remove_intermediary_models(model_hashes):
 
     models.delete()
 
-    log_model_hashes = '\n\t- '.join(filtered_model_hashes)
-    logger.info(f'Remove intermediary models : \n\t- {log_model_hashes}')
+    if filter_model_hashes:
```

Comment:
`filtered_model_hashes`

Reply:
wow this must be my most embarrassing PR ever :sweat_smile: